lib/config/presets/internal/monorepo.ts

```diff
@@ -292,6 +292,7 @@ const patternGroups = {
   'aws-java-sdk-v2': '^software.amazon.awssdk:',
   babel6: '^babel6$',
   clarity: ['^@cds/', '^@clr/'],
+  'datatables.net': ['^datatables.net'],
```

Comment:
can't we do this via sourceUrl instead?

Reply:
Good idea.  I’ll have a look tomorrow.

Reply:
Moving it to `orgGroups` worked the same and is a better approach. 👍🏻 

---



However, I'm don't know how this can be included.  As I've seen `@types/` being included in PRs for other packages.
```json
"packageName": "@types/datatables.net",
"sourceUrl": "https://github.com/DefinitelyTyped/DefinitelyTyped",
```